examples/draw_climatology_map_MISR_AOD.py

```diff
@@ -0,0 +1,34 @@
+import ocw.dataset as ds
+import ocw.data_source.local as local
+import ocw.dataset_processor as dsp
+import ocw.plotter as plotter
+
+import numpy as np
+import numpy.ma as ma
+
+
+''' data source: https://dx.doi.org/10.6084/m9.figshare.3753321.v1'''
+dataset = local.load_file('/home/huikyole/climate/examples/AOD_monthly_2000-MAR_2016-FEB_from_MISR_L3_JOINT.nc', 
```

Comment:
This relative path needs to be replaced ideally by a remote, publicly available one @huikyole